src/gc/unix/gcenv.unix.cpp

```diff
@@ -157,6 +160,60 @@ FOR_ALL_NUMA_FUNCTIONS
 #undef PER_FUNCTION_BLOCK
 #endif // HAVE_NUMA_H
 
+#if defined(__linux__) && HAVE_NUMA_H
```

Comment:
A nit - if you moved the `#endif` above after the end of your added code, this `#if defined(__linux__) && HAVE_NUMA_H` / `#elif HAVE_NUMA_H` would be a bit simpler - just `#ifdef __linux__` / `#else`.

Reply:
Done.